assets/dev/js/editor/controls/base-data.js

```diff
@@ -236,16 +236,15 @@ ControlBaseDataView = ControlBaseView.extend( {
 	 * @returns {*}
 	 */
 	preparePlaceholderForChildren: function() {
-		const parentValue = this.getResponsiveParentView()?.preparePlaceholderForChildren(),
-			cleanValue = this.getCleanControlValue();
+		const cleanValue = this.getCleanControlValue(),
+			parentValue = this.getResponsiveParentView()?.preparePlaceholderForChildren(),
+			placeholder = this.model.get( 'placeholder' );
 
 		if ( cleanValue instanceof Object ) {
-			return Object.keys( cleanValue ).length ?
-				Object.assign( {}, parentValue, cleanValue ) :
-				parentValue;
+			return Object.assign( {}, placeholder, parentValue, cleanValue );
```

Comment:
`parentValue` may be null

Reply:
> `parentValue` may be null

The main goal of the method is to prepare the placeholder for the child responsive control, for example - when we want the placeholder for the mobile control, we call this method on the tablet control so it can return the right placeholder. The idea behind the `Object.assign` is a hierarchy, a priority of what should come first as a placeholder. The first in priority, is the `cleanValue`, which is the value without defaults and empty properties. The second is the parent's value and at the last priority - the placeholder. Sometimes the `prarentValue` can be null, but there can be other values available such as `placeholder` and `cleanValue`. When none of them is available, `null` is the appropriate prepared placeholder.